src/Storages/System/StorageSystemTables.cpp

```diff
@@ -244,6 +244,9 @@ class TablesBlockSource : public SourceWithProgress
                         // lifetime_bytes
                         if (columns_mask[src_index++])
                             res_columns[res_index++]->insertDefault();
+
+                        if (columns_mask[src_index++])
+                            res_columns[res_index++]->insertDefault();
```

Comment:
A comment needed, all such `if`'s above have a comment for inserted column name, this one needs too.